scripts/tests/scripts_regression_tests.py

```diff
@@ -610,6 +610,12 @@ def test_wait_for_test_no_wait(self):
         self.simple_test(self._testdir_unfinished, expected_results, "-n")
 
     ###########################################################################
+    def test_wait_for_test_timeout(self):
```

Comment:
We need to add the --timeout flag to the scripts_regression_tests.py command line as well.